tests/idris2/total/total027/expected

```diff
@@ -0,0 +1,79 @@
+1/1: Building dataSameTotality (dataSameTotality.idr)
+Error: X is not total, not strictly positive
+
+dataSameTotality:5:1--6:22
+ 5 | data X where
+ 6 |   MkX : (X -> X) -> X
+
+------
+1/1: Building recordSameTotality (recordSameTotality.idr)
+Error: X is not total, not strictly positive
+
+recordSameTotality:4:1--7:13
+ 4 | total
+ 5 | record X where
+ 6 |   constructor MkX
+ 7 |   f : X -> X
+
+------
+1/1: Building dataDifferentTotality (dataDifferentTotality.idr)
+Error: Data X has been forward-declared with totality `partial`, cannot change to `total`
+
+dataDifferentTotality:4:1--6:22
+ 4 | partial
+ 5 | data X where
+ 6 |   MkX : (X -> X) -> X
+
+Possible solutions:
+ - Use the same totality modifiers
+ - Remove the totality modifier from the declaration
+ - Remove the totality modifier from the definition
+------
+1/1: Building recordDifferentTotality (recordDifferentTotality.idr)
+Error: Data Main.X has been forward-declared with totality `partial`, cannot change to `total`
```

Comment:
In this error, unlike the others, the name is printed along with the namespace. I'm not sure if this is the correct behavior, and how best to fix it

Reply:
I think we can leave that be for now